shared_model/interfaces/common_objects/impl/peer.cpp

```diff
@@ -6,6 +6,7 @@
 #include "interfaces/common_objects/peer.hpp"
 
 #include "cryptography/public_key.hpp"
+#include "logger/logger.hpp"
```

Comment:
Please remove and probably use approach with `value_or`

Reply:
What do you mean by "approach with `value_or`"? Previously in this PR we have agreed to show only the fact of presence of a certificate (true/false), and not the whole certificate.

I surely can use something like
```
bool(tlsCertificate()) ? "true" : "false"
```
but I'm not sure it would be as readable.

Another option would be to add an overload - `PrettyStringBuilder::append(bool value)`, which would do serialization by itself.

Reply:
Overload option seems the best to me.